tests/test_package_meta.py

```diff
@@ -17,3 +17,24 @@ def test_package_metadata_version():
 
 def test_hashable():
     hash(PackageMetadata(name="test", version="1.0"))
+
+
+def test_package_metadata_extra_field():
+    pkg = {
+        "name": "test",
+        "version": "1.0",
+        "maintainer": "bob",
+        "extra_field_that_is_definitely_not_in_the_model": False,
+    }
+
+    try:
+        p = PackageMetadata(**pkg)
```

Comment:
@aganders3 I am confused, so with the fix it should not fail with extra fields? So why is the test checking that it fails. Or am I missing something 🤔

Reply:
You're right - the try/except is a little ugly but is basically rewriting whatever error would come up into an `AssertionError` for consistency with pytest expectations.

Then after parsing succeeds the other assertions check known fields are present/correct and the extra field is ignored/dropped. Does that make sense?

Reply:
Yep :)